applications/API/filters.py

```
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Pylint (3.3.7)

[warning] Unknown option value for '--disable', expected a valid pylint message and got 'long-suffix'

(W0012)